src/applications/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ option(ENABLE_calibrate "Build camera calibration" OFF)
 option(ENABLE_view "Build basic viewer" ON)
 option(ENABLE_corner "Build corner detector" OFF)
 option(ENABLE_dualCamTransform "Build event to frame transform" OFF)
+option(ENABLE_vAnnotation "Build application to automatically annotate events" OFF)
```

Comment:
remove vAnnotation from event-driven and put it as an external module with all the code for learning object recognition